iib/web/models.py

```diff
@@ -1006,13 +1037,22 @@ def from_json(cls, kwargs, batch=None):
             raise ValidationError(f'"operators" should be a non-empty array of strings')
 
         cls._from_json(
-            request_kwargs, additional_required_params=['operators', 'from_index'], batch=batch
+            request_kwargs,
+            additional_required_params=['operators', 'from_index'],
+            additional_optional_params=['build_tags'],
+            batch=batch,
         )
 
         request_kwargs['operators'] = [Operator.get_or_create(name=item) for item in operators]
 
         request = cls(**request_kwargs)
         request.add_state('in_progress', 'The request was initiated')
+
+        build_tags = request_kwargs.pop('build_tags', [])
```

Comment:
Same, you have a function for this. `add_build_tag`

Reply:
hmm, I don't understand